docs/chainOps/k8s/tutorials/betanet.md

```diff
@@ -147,28 +177,49 @@ e.g.:
 ```
 rake "validator:keys:public[my-cluster,aws,sifnode]"
 ```
+Note: This requires jq JSON processor if not installed install with ```sudo apt-get install jq```
 
-3. Stake:
-
+3. Run the following command to become a validator:
 ```
-rake "validator:stake[<chain_id>,<moniker>,<amount>,<gas>,<public_key>,<node_rpc_address>]"
```

Comment:
@sahilpant The entry point must be the rake task. If there's something you don't like about the rake task, please update and include in your PR.

Reply:
actually I was not able to do the stake transaction with rake and most of the people were also facing this issue and were not able to do the transaction and did it with sifnodecli, just like I did. So, it would be great if we one the first place provide the command that actually works.

Reply:
Thanks Sahil, were you ever able to stake with rake?  If so, what is the command you used?  Please add that to your PR

Reply:
@sahilpant Rake works just fine. There's a reason why we have a single entry point. You need to provide more details around what exactly didn't work.

Reply:
rake "validator:stake[sifchain-testnet,my-node,1000000000000000000rowan,0.5rowan,sifvalconspub1zcjduepqzts0caumeczq0l23j4fg
9hq5m395uzujzs5h2tq8mgt9kx6c5upsgp75ps,tcp://rpc-testnet.sifchain.finance:80]"

I entered this command many a times but every time my txn gets rejected but the gas gets deducted and it shows on the explorer also.
https://discord.com/channels/769209144515100693/769209366212640808/833717541092327474
This is one of the txn that got cancelled you can see around this block height there were many txn from this address which were got cancelled cause I rand this command with different values of gas and staking rowans.

Reply:
Thanks @sahilpant - the ideal solution is to update the rake task to allow a `gas` and a `gas_prices` argument. See here:

Reply:
ok got it !!! I have made changes in this file itself. Let me know if this is good to go or not.

Reply:
@sahilpant Almost there :) You also need to:

* update the rake file (as per my example).
* update the table beneath the command.

and then that should be it. Thanks for this.

Reply:
did some changes